active-rfcs/0000-recursive-components.md

```diff
@@ -0,0 +1,150 @@
+-   Start Date: Date: 2021-12-15
+-   Reference Issues: https://github.com/withastro/rfcs/discussions/45
+-   Implementation PR:
+
+# Summary
+
+Allow a way for astro components to render themselves recursively.
```

Comment:
There's a lot of good content in this section but I think most of it belongs in the __Motiviation__ section. This sentence is good, and maybe add one or two explaining that `Astro.self` is the proposed solution.